src/pages/index.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import styles from "./index.module.css";
+import Card from "../components/Card";
+
+function Home() {
+  const context = useDocusaurusContext();
+  const { siteConfig = {} } = context;
+
+  return (
+    <Layout title="Homepage" description="Avalanche Documentation">
+      <main>
+        <br/>
+        <h1 align="center" style={{ fontWeight: '750' }}>Welcome to Avalanche Docs</h1>
+        <section className={styles.features}>
+          <div className="container">
+            <div className="row cards__container">
+              <Card
+                to="subnets/build-first-subnet"
+                header={{
+                  label: "🚀 Launch Your First Subnet",
+                }}
+                body={{
+                  label:
+                    "Start your Subnet development journey by creating a subnet in under five minutes",
+                }}
+              />
+
+              <Card
+                to="intro"
+                header={{
+                  label: "🔺 Learn about Avalanche",
+                }}
+                body={{
+                  label:
+                    "Discover how Subnets and Avalanche Consensus are revolutionizing Web3",
+                }}
+              />
+
+              <Card
+                to="nodes"
+                header={{
+                  label: "😎 Become a validator",
+                }}
+                body={{
+                  label:
+                    "Join Avalanche's Proof-of-Stake protocol to help secure the network and earn rewards",
+                }}
+              />
+
+              <Card
+                to="apis/avalanchego"
+                header={{
+                  label: "💻 View Avalanche APIs",
+                }}
+                body={{
+                  label:
+                    "Access low-level protocol interfaces to build your custom dapp",
+                }}
+              />
+
+              <Card
+                to="dapps/launch-your-ethereum-dapp"
+                header={{
+                  label: "🛠️ Launch Your Dapp on Avalanche",
+                }}
+                body={{
+                  label:
+                    "Learn everything you need to deploy an EVM-compatible smart contract",
+                }}
+              />
+
+              <Card
+                to="https://core.app"
```

Comment:
Not sure about this link to core.app, as it does not do an explanation what Core is. https://support.avax.network/en/collections/3391518-core is not a good choice, either.

Reply:
I was thinking we wanted a link to each of our most important topics, so I wanted to have one for Core. Not sure what the best page that represents Core is, but that one seemed like the best I could find.

Reply:
going to leave as is for now, happy to update later if we have more consensus